webdriver/tests/support/fixtures.py

```diff
@@ -262,15 +262,20 @@ def create_dialog(dialog_type, text=None):
 
         assert isinstance(text, basestring), "`text` parameter must be a string"
 
+        dialog_params = "text"
+        if dialog_type == "prompt":
+            # For prompt() dialogs, add a value for the 'default' argument
+            dialog_params += ", ''"
```

Comment:
Mind, completely moving this into the script code so that we can get rid of the call to `format()`?